nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-settings.js

```diff
@@ -1925,6 +1931,872 @@
+                if (target.hasClass('view-flow-analysis-rule')) {
+                    nfFlowAnalysisRule.showDetails(flowAnalysisRuleEntity);
+                } else if (target.hasClass('flow-analysis-rule-usage')) {
+                    // close the settings dialog
+                    $('#shell-close-button').click();
+
+                    // open the documentation for this flow analysis rule
+                    nfShell.showPage('../nifi-docs/documentation?' + $.param({
+                        select: flowAnalysisRuleEntity.component.type,
+                        group: flowAnalysisRuleEntity.component.bundle.group,
+                        artifact: flowAnalysisRuleEntity.component.bundle.artifact,
+                        version: flowAnalysisRuleEntity.component.bundle.version
+                    })).done(function () {
+                        nfSettings.showSettings();
+                    });
+                }
+            }
+        });
+
+        // wire up the dataview to the grid
+        flowAnalysisRulesData.onRowCountChanged.subscribe(function (e, args) {
+            flowAnalysisRulesGrid.updateRowCount();
+            flowAnalysisRulesGrid.render();
+        });
+        flowAnalysisRulesData.onRowsChanged.subscribe(function (e, args) {
+            flowAnalysisRulesGrid.invalidateRows(args.rows);
+            flowAnalysisRulesGrid.render();
+        });
+        flowAnalysisRulesData.syncGridSelection(flowAnalysisRulesGrid, true);
+
+        // hold onto an instance of the grid
+        $('#flow-analysis-rules-table').data('gridInstance', flowAnalysisRulesGrid).on('mouseenter', 'div.slick-cell', function (e) {
+            var errorIcon = $(this).find('div.has-errors');
+            if (errorIcon.length && !errorIcon.data('qtip')) {
+                var taskId = $(this).find('span.row-id').text();
+
+                // get the task item
+                var flowAnalysisRuleEntity = flowAnalysisRulesData.getItemById(taskId);
+
+                // format the errors
+                var tooltip = nfCommon.formatUnorderedList(flowAnalysisRuleEntity.component.validationErrors);
+
+                // show the tooltip
+                if (nfCommon.isDefinedAndNotNull(tooltip)) {
+                    errorIcon.qtip($.extend({},
+                        nfCommon.config.tooltipConfig,
+                        {
+                            content: tooltip,
+                            position: {
+                                target: 'mouse',
+                                viewport: $('#shell-container'),
+                                adjust: {
+                                    x: 8,
+                                    y: 8,
+                                    method: 'flipinvert flipinvert'
+                                }
+                            }
+                        }));
+                }
+            }
+
+            var bulletinIcon = $(this).find('div.has-bulletins');
```

Comment:
Thanks for the update to remove the bulletins from the Flow Analysis Rules, however, there are still some remnants here.